lib/notifications.dart

```diff
@@ -283,14 +284,16 @@ class NotificationDisplayManager {
   }
 
   static void _onMessageFcmMessage(MessageFcmMessage data, Map<String, dynamic> dataJson) {
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
```

Comment:
On the translations.md documentation it says to use `ZulipLocalizations.of(context)` to access the localization object, but I haven't been able to get the `BuildContext` here. Is there a better way to handle this @gnprice?

Reply:
Yeah. This is a context where there's no associated widget in Flutter's trees, so there's no relevant BuildContext and we should just use this global value.